src/components/badge-list/index.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames'
+import Badge from 'components/badge'
+import { BadgeProps } from 'components/badge/types'
+import Tooltip from 'components/tooltip'
+import s from './badge-list.module.css'
+
+export interface Badge {
+	text: string
+	icon?: React.ReactElement
+	tooltip?: string
+}
+
+interface BadgeListProps {
+	badges: Array<Badge>
+	className?: string
+	size?: BadgeProps['size']
+	type?: BadgeProps['type']
+	color?: BadgeProps['color']
+}
+
+export default function BadgeList({
+	badges,
+	className,
+	size = 'medium',
+	color = 'neutral',
+	type = 'filled',
+}: BadgeListProps) {
+	return (
+		<ul className={classNames(s.badgeList, className)}>
```

Comment:
💭 One approach I really like with these list wrapper components is for them to just render children. I find the flexibility there making them much easier to reuse. [Here's one example.](https://github.com/hashicorp/dev-portal/blob/76355ca7a76e69d24e6e32d42bc5a046a98e5687/src/components/navigation-header/components/product-page-content/index.tsx#L44-L59) Not necessary to act on, but just sharing as food for thought.

Reply:
Another great example of rendering children here https://github.com/hashicorp/dev-portal/blob/main/src/components/cards-grid-list/index.tsx

Reply:
@BrandonRomano have you given this feedback further thought?

Reply:
For this PR though; I don’t think I’m going to change it, although I’ve thought about the suggestion!

There are a few things that I want to codify as rules of this list component onto badges
- I want to be able to codify `size` and `type` always being the same at the list level.
- I want a badge to be able to specify its `tooltip`; this is something the consumer would have to wrap in a composition based approach, whereas with a props approach we can hide the internals that behavior.

That being said, I broke out a thread in slack to talk about this pattern from a higher level. Would love to hear your thoughts in there: https://hashicorp.slack.com/archives/C02T80JTNS2/p1677687494767269